src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -519,6 +520,8 @@ private static GeneratableElement getExampleFor(ModElement modElement, boolean u
 					Color.red, new Procedure("condition1")));
 			components.add(new Label("text2", 100, 150, new StringProcedure(!_true ? "string2" : null, "fixed value 2"),
 					Color.white, new Procedure("condition4")));
+			components.add(new Label("text3", 100, 150, new StringProcedure(null, "fixed value 3"),
+					Color.white, new Procedure("condition1"), IAnchorableElement.AnchorPoint.TOP_LEFT));

```

Comment:
The AnchorPoint value should be taken randomly instead of a fixed value, as you are currently one part of your code template only (and you can also use the previous label)

Reply:
Should've thought of that...
I'm not sure what you mean by "use the previous label"
I have to leave the original tests as they are to keep that constructor tested, which is stull used in custom GUIs!?

Reply:
> I have to leave the original tests

2 labels already exist, so you can modify one of the labels and keep the other one to test the original constructor/the constructor without an AnchorPoint.